RFCs/RFC_2.md

```diff
@@ -0,0 +1,48 @@
+# RFC: Branch and PR naming and tagging convention
+
+**Primary author(s)**: Giovan Angela
+
+**Created**: 10-10-2022
+
+**Last updated**: 10-10-2022
+
+**Approver(s)**:
+
+| Name | yes/not yet | date |
+| ---- | ----------- | ---- |
+|      | [ ]         |      |
+
+## Overview
+<!-- If someone only reads this far, what do you want them to know? -->
+Use one convention when naming branches or PR's.
+
+## Goal
+<!-- What problems are you trying to solve? What problems are you not trying to solve? -->
+The goal is to have one naming convention for PR's and branches such that it looks nice in git(hub).
+
+## Motivation
+<!-- What is the current state of the world? Why is this change being proposed? -->
+There are several types of names beings used without a clear structure. Some even use the name to indicate if it is an enchancement, feature, bugfix, etc... This does not look nice or organised.
+
+## Design
+<!-- How will this be implemented? -->
+Use the Jira board issue title as PR and branch title. For example this fictive issue for Summa (SUM-415) with description "Add documentation for undocumented code" will have a branch named "sum-415-add-documentation-for-undocumented-code" and it's PR will should be named "SUM-415 Add documentation for undocumented code". For branches and PRs that do not have an issue in Jira -> create a ticket in Jira.
```

Comment:
How are we going to enforce this behaviour?

Reply:
1. https://github.com/varias-io/standards/pull/2#discussion_r1016722457
2. Lead-dev should ideally generally gatekeep any releases / PR's of project
3. Document, and enforce, any format via documentation and setup individually
4. Add branch-protection rules (in jira, git-hooks or github)

Reply:
```suggestion
Use the Jira board issue title as PR and branch title. For example this fictive issue for Summa (SUM-415) with description "Add documentation for undocumented code" will have a branch named "sum-415/add-documentation-for-undocumented-code" and it's PR will should be named "SUM-415 Add documentation for undocumented code". For branches and PRs that do not have an issue in Jira -> create a ticket in Jira.
```

means no ambiguity in title & ticket number, and easier automation if you ever need it.